sympy/geometry/tests/test_ellipse.py

```diff
@@ -1,14 +1,30 @@
 from __future__ import division
 
-from sympy import Dummy, Rational, S, Symbol, symbols, pi, sqrt, oo
+from sympy import Rational, S, Symbol, symbols, pi, sqrt, oo, Point2D
 from sympy.core.compatibility import range
 from sympy.geometry import (Circle, Ellipse, GeometryError, Line, Point, Polygon, Ray, RegularPolygon, Segment,
                             Triangle, intersection)
-from sympy.integrals.integrals import Integral
-from sympy.utilities.pytest import raises, slow
+from sympy.utilities.pytest import raises
 from sympy import integrate
 from sympy.functions.special.elliptic_integrals import elliptic_e
 
+def test_object_from_equation():
+    from sympy.abc import x, y, a, b
+    assert Circle(x ** 2 + y ** 2 + 3 * x + 4 * y - 8) == Circle(Point2D(S(-3) / 2, -2),
+                                                                                      sqrt(57) / 2)
+    assert Circle(x ** 2 + y ** 2 + 6 * x + 8 * y + 25) == Circle(Point2D(-3, -4), 0)
+    assert Circle(a ** 2 + b ** 2 + 6 * a + 8 * b + 25, x='a', y='b') == Circle(Point2D(-3, -4), 0)
+    assert Circle(x ** 2 + y ** 2 - 25) == Circle(Point2D(0, 0), 5)
+    assert Circle(x ** 2 + y ** 2) == Circle(Point2D(0, 0), 0)
+    assert Circle(a ** 2 + b ** 2, x='a', y='b') == Circle(Point2D(0, 0), 0)
+    assert Circle(x ** 2 + y ** 2 + 6 * x + 8) == Circle(Point2D(-3, 0), 1)
+    assert Circle(x ** 2 + y ** 2 + 6 * y + 8) == Circle(Point2D(0, -3), 1)
```

Comment:
Perhaps a test where all the coefficients are multiplied by a constant would be good.

Reply:
@smichr Done that now. Please take a look.